lib/active_merchant/billing/gateways/netbanx.rb

```diff
@@ -132,13 +132,7 @@ def add_credit_card(post, credit_card, options = {})
 
       def add_invoice(post, money, options)
         post[:amount] = amount(money)
-        post[:currencyCode] = options[:currency] if options[:currency]
```

Comment:
do we also need the currency code in the `add_invoice` in addition to the `add_payment`?

Reply:
@elfassy No, it only needs to be in add_payment